Contest/Assignment/src/org/togetherjava/event/elevator/elevators/Elevator.java

```diff
@@ -37,35 +40,74 @@ public Elevator(int minFloor, int floorsServed, int currentFloor) {
         this.minFloor = minFloor;
         this.currentFloor = currentFloor;
         this.floorsServed = floorsServed;
+        this.destinations = new ArrayList<>();
     }
 
+    /**
+     * The unique ID of the elevator.
+     *
+     * @return the unique ID of the elevator
+     */
```

Comment:
the method already has javadoc through the interface its coming from (`@Override`)